plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -2481,6 +2486,13 @@ public void renameMaterializedView(ConnectorSession session, SchemaTableName sou
         catalog.renameMaterializedView(session, source, target);
     }
 
+    private static void commitUpdateWithExtraMetadata(SnapshotUpdate<?> update, ConnectorSession session)
```

Comment:
Please add this kind of handling also to the operations from `io.trino.plugin.iceberg.catalog.AbstractTrinoCatalog`.

Reply:
Applied the same handling for materialized view creation (which does create a snapshot). Other operations in `AbstractTrinoCatalog` are related to adding/updating table or column comments (which do not create a new snapshot).